msctl

```diff
@@ -455,6 +455,24 @@ enableWorld() {
   setMSCSValue "$1" "mscs-enabled" "true"
 }
 
+# ---------------------------------------------------------------------------
+# Import a world.
+#
+# @param 1 The location of the world to import.
+# @param 2 The name of the new imported world.
+# @param 3 The port of the new imported world.
+# @param 4 The IP address of the new imported world.
+# ---------------------------------------------------------------------------
+importWorld() {
+  local NAME MCUSER MCGROUP
+  mkdir -p "$WORLDS_LOCATION/$2"
+  cp -R $1/* $WORLDS_LOCATION/$2
+  chown $(id -un):$(id -gn) -R $WORLDS_LOCATION/$2
+  NAME=$(getServerPropertiesValue $2 'level-name')
+  mv $WORLDS_LOCATION/$2/$NAME $WORLDS_LOCATION/$2/$2
```

Comment:
Might want to add an exists check before calling `mv`

Reply:
That is a good idea.  I just committed the change to the PR